payments.md

```diff
@@ -58,21 +67,30 @@ payment as a resource represents following information
 ### Headers
 	POST /payment
 	cid: 05e230bf-a9cf-4b31-bc54-d3a995f62526
+	Content-Type: application/vnd.api+json
+	Accept: application/vnd.api+json
 			
-### Request (application/json)
+### Request
+	
   ```json
   {
+        "jsonapi": {
+            "version": "1.0"
+        },
         "payment": {
 		"contractAccount": "600134652",
```

Comment:
Please change to contract-account-number and update the sample value as 600123456.
Also update card to card-details and updated sample value as an encrypted string.